src/main.rs

```diff
@@ -118,6 +130,16 @@ impl Args {
         let other = Other {
             addr: self.addr,
             jobs: self.jobs,
+            port_range: if self.port_min == 0 || self.port_max == 0 {
+                None
+            } else {
+                Some(self.port_min..self.port_max)
+            },
```

Comment:
`port_min` of `0` is a valid value

If `port_max` is `0` we should allow `port_min..`, i.e. anything higher than `port_min`